mesos/mesos.go

```diff
@@ -121,6 +137,7 @@ func (m *Mesos) GetMetricTypes(cfg plugin.ConfigType) ([]plugin.MetricType, erro
 
 		for _, key := range agent_stats {
 			namespace := core.NewNamespace(pluginVendor, pluginName, "agent").
+				AddStaticElement("task").
```

Comment:
i have never seen such case in general ... but you are right. 
I can rename it to executor/<fid>/<eid>